docusaurus/src/remark/docsHeaderDecoration.js

```diff
@@ -0,0 +1,111 @@
+const fetch = require("node-fetch");
+const visit = require("unist-util-visit");
+
+const registry_url =
+  "https://connectors.airbyte.com/files/generated_reports/connector_registry_report.json";
+let registry = [];
+
+const plugin = () => {
+  let registryFetcher = fetchCatalog();
+
+  const transformer = async (ast, vfile) => {
+    if (!isDocsPage(vfile)) return;
+
+    const pathParts = vfile.path.split("/");
+    const connectorName = pathParts.pop().split(".")[0];
+    const connectorType = pathParts.pop();
+    const dockerRepository = `airbyte/${connectorType.replace(
+      /s$/,
+      ""
```

Comment:
@evantahler Why are we generically removing any `s` at the end of a connector type?

Reply:
Ah I think I get it, this is only here to make `destinations` -> `destination`, `sources` -> `source`.